src/main/java/org/spongepowered/common/command/SpongeCommandFactory.java

```diff
@@ -560,9 +561,15 @@ private static CommandSpec createSpongePluginsCommand() {
                 .arguments(optionalWeak(literal(Text.of("reload"), "reload")), optional(plugin(Text.of("plugin"))))
                 .executor((src, args) -> {
                     if (args.hasAny("reload") && src.hasPermission("sponge.command.plugins.reload")) {
-                        src.sendMessage(Text.of("Sending reload event to all plugins. Please wait."));
                         Sponge.getCauseStackManager().pushCause(src);
-                        SpongeImpl.postEvent(SpongeEventFactory.createGameReloadEvent(Sponge.getCauseStackManager().getCurrentCause()));
+                        if (args.hasAny("plugin")) {
+                            PluginContainer plugin = args.<PluginContainer>getOne("plugin").get();
```

Comment:
This can crash the command, can you check and if it's not, return an appropriate response to the sender?

Reply:
This shouldn't be crashing the command, as the `plugin` argument is parsed through `GenericArguments.plugin`. If the type isn't a `PluginContainer` then there are other issues at play.

Reply:
Oh, duh, it's behind the `hasAny` call. But, given that it's not checking if the plugin is actually there, would that still pose an issue?

Reply:
No. They use the same key. I believe dual has a PR open for an API method to get it directly for cases when you know the argument is present, so it's a standard thing here.

Reply:
This is fine with `hasAny` then `getOne`, if it crashes, it's indicative of an issue.

It may be slightly better to do `getOne` then use the optional for performance reasons, rather than having two calls to the context, but that's splitting hairs. It's fine as is.